test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -69,6 +69,22 @@ Describe 'Basic Job Tests' -Tags 'CI' {
             $ProgressMsg[0].StatusDescription | Should -BeExactly 2
         }
 
+        It 'Can use the user specified working directory parameter' {
+            $tempPath = [System.IO.Path]::GetTempPath()
+            # In the script block we use join path to normalize the path string
+            $job = Start-Job -ScriptBlock { Join-Path $pwd "" } -WorkingDirectory $tempPath | Wait-Job
+            $jobOutput = Receive-Job $job
+            $jobOutput | Should -BeExactly $tempPath
+        }
+
+        It 'Throws an error message if the working directory parameter points to an invalid path' {
+            $invalidPaths = @(""," ", "this is an invalid path")
```

Comment:
Please use testcases pattern instead. The benefit of that is if the first one fails, it still executed the second case. Please see https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#parameter-generation

Reply:
Yeah this approach is way cleaner. Fixed in 62a7b0c71ac60747f84ffba8f523ff0004fd17c9